academic/import_bibtex.py

```diff
@@ -139,12 +139,25 @@ def parse_bibtex_entry(
     if "keywords" in entry:
         page.fm["tags"] = clean_bibtex_tags(entry["keywords"], normalize)
 
-    if "url" in entry:
-        page.fm["url_pdf"] = clean_bibtex_str(entry["url"])
-
     if "doi" in entry:
         page.fm["doi"] = clean_bibtex_str(entry["doi"])
 
+    links = []
+    if "archiveprefix" in entry and "eprint" in entry:
+        if entry["archiveprefix"].lower() == "arxiv":
```

Comment:
Why are you nesting this if statement? Python supports short-circuit evaluation.

Reply:
Also, we can rewrite multiple `in` checks using `all`

Reply:
Thanks, made the changes.